### PR TITLE
feat(oracle-v3): add a new env variable `MEMBER_PRIV_KEY_FILE`

### PR DESCRIPTION
**Pull request by selfuryon**
The same as https://github.com/lidofinance/lido-oracle/pull/327, but for oracle-v3

### CODE REVIEW

File: src/variables.py [hunk 1]
```diff
@@ -10,6 +10,12 @@
 # - Account -
 ACCOUNT = None
 MEMBER_PRIV_KEY = os.getenv('MEMBER_PRIV_KEY')
+
+MEMBER_PRIV_KEY_FILE = os.getenv('MEMBER_PRIV_KEY_FILE')
+if MEMBER_PRIV_KEY_FILE is not None:
```
Comment (skhomuti) [inline src/variables.py:15]:
```
if not os.path.exists(MEMBER_PRIV_KEY_FILE): 
    raise ValueError(f"File {MEMBER_PRIV_KEY_FILE} is not found")
```

Comment (F4ever) [reply src/variables.py:15]: Yeah, let's add more details, like "provided path is not valid" or smth similar

**Comment by F4ever** [comment]
Thanks for PR! We will add this feature to the next pre-release.

File: src/variables.py (hunk 1 above)
Comment (selfuryon) [reply src/variables.py:15]: Ok, I'll add today that

**Pull request merged**